index.d.ts

```diff
@@ -1,8 +1,10 @@
 declare module 'athena-express' {
-    import * as aws from 'aws-sdk';
+    import { S3 } from '@aws-sdk/client-s3';
+    import { Athena } from '@aws-sdk/client-athena';
     interface ConnectionConfigInterface {
-        aws: typeof aws;
-        s3: string;
+        athena: typeof Athena,
+        s3: typeof S3,
+        s3Bucket: string;
```

Comment:
this is a breaking change. Need advice on the right naming of the `s3` client in case you want to keep the original `s3` attribute for the bucket